src/components/AssetsComponent.jsx

```diff
@@ -104,50 +96,31 @@ export default class AssetsComponent extends Component {
     const { status } = this.props;
     const totalPages = this.handlePageTotal();
     const currentAssets = `page_${this.props.activePage}`;
+    const showPaginator = totalPages > 1;
     const showFilter = !isEmpty(this.props.assetsList[currentAssets] || assets);
 
-    const contentTitle = status ? `${status.toLocaleString()} Assets` : 'Assets';
-
     return (
-      <NavBarComponent title="Assets">
-        <div className="assets-list">
-          <div id="page-heading-section">
-            <Header as="h1" id="page-headings" floated="left" content={contentTitle} />
-            <Divider id="assets-divider" />
-            {showFilter && (
-              <FilterButton
-                activePage={this.props.activePage}
-                limit={this.state.limit}
-                selected={this.props.selected}
-                filterAction={this.props.getAssetsAction}
-                disabled={this.props.isLoading}
-              >
-                <React.Fragment>
-                  <FilterComponent
-                    index={0}
-                    option={this.props.filterData[0]}
-                    selected={this.props.selected}
-                    filterSelection={this.props.filterSelection}
-                  />
-
-                  <FilterComponent
-                    index={1}
-                    option={this.props.filterData[1]}
-                    selected={this.props.selected}
-                    filterSelection={this.props.filterSelection}
-                  />
-                </React.Fragment>
-              </FilterButton>
-            )}
-          </div>
-          <AssetsTableContent
+      <Fragment>
+        {showFilter && (
+          <Filter
             activePage={this.props.activePage}
-            assets={this.props.assetsList[currentAssets] || assets}
-            errorMessage={this.props.errorMessage}
-            hasError={this.props.hasError}
-            isLoading={this.props.isLoading}
-            status={status}
+            limit={this.state.limit}
+            filterData={this.props.filterData}
+            selected={this.props.selected}
+            filterSelection={this.props.filterSelection}
+            filterAction={this.props.getAssetsAction}
+            disabled={this.props.isLoading}
           />
+        )}
+        <AssetsTableContent
+          activePage={this.props.activePage}
+          assets={this.props.assetsList[currentAssets] || assets}
+          errorMessage={this.props.errorMessage}
+          hasError={this.props.hasError}
+          isLoading={this.props.isLoading}
+          status={status}
+        />
+        {showPaginator && (

```

Comment:
`showPaginator` is not needed. `PaginationComponent` renders null when the `totalPages <= 1`. Check the logic [here](https://github.com/AndelaOSP/art-dashboard/blob/develop/src/components/common/PaginationComponent.jsx#L14)